### ISSUE TITLE
Brazillian Portuguese Translation

### ISSUE THREAD

**Issue by Hirien** [steverichey/DMLocalizer#15]
Language: Brazillian Portuguese (PT-BR)
Translator: Gabriel B. Manduca

```
A game by: Um jogo por
Powered by: Desenvolvido com
Press left or right to start: Pressione esquerda ou direita para começar
Release left and right: Solte esquerda e direita
to start: para começar
to erase: para apagar
Touch left or right side to start: Toque no lado esquerdo ou direito para começar
Release both sides: Solte ambos os lados
Fill bar to erase save data: Preencha a barra para apagar os dados salvos
Erasing save data: Apagando os dados salvos
attempts: tentativas
distance: distância
seconds: segundos
medals: medalhas
levels: níveis
coins: moedas
trophies: troféus
zenith: zênite
zeniths: zênites
the end: fim
trophy unlocked: troféu desbloqueado
paused: pausado
time: tempo
Hold left or right to restart: Pressione esquerda ou direita para reiniciar
Touch screen to restart: Toque a tela para reiniciar
```

Localized images:
![pt-br_dont-move](https://cloud.githubusercontent.com/assets/11844297/7036799/2f16d14c-dd6f-11e4-8abf-e3161aa46456.png)
![pt-br_level_up](https://cloud.githubusercontent.com/assets/11844297/7036803/39ec84b8-dd6f-11e4-8a0f-9d7bd8de66f2.png)
![pt-br_zenith](https://cloud.githubusercontent.com/assets/11844297/7036805/3fac95d2-dd6f-11e4-802e-645f235096a4.png)
![pt-br_gameover](https://cloud.githubusercontent.com/assets/11844297/7036806/43e2565a-dd6f-11e4-92a9-a9cacb731e53.png)

**Comment by Hirien** [comment]
I didn't mess with capitalization because I assumed most of the text strings were meant to be lowercase only. Is that correct?

**Comment by steverichey** [comment]
Yep, that looks great, thank you!

**Comment by Hirien** [comment]
No problem, Steve! Looking forward to play the game :)!

**Comment by steverichey** [comment]
Implemented in `f1cd5f70`
